onboarding/schema.py

```diff
@@ -392,8 +404,10 @@ class OnboardingSessionInfo(object):
     A mixin class defining all onboarding-related queries.
     """
 
-    onboarding_step_1 = OnboardingStep1V2Info.field()
-    onboarding_step_3 = OnboardingStep3Info.field()
+    onboarding_step_1 = graphene.Field(OnboardingStep1V2Info)
```

Comment:
What's the consequence of changing this type if we're not getting rid of OnboardingStep1V2Info here?

Reply:
So we're actually not changing the type itself, we're just changing it so that we provide a custom resolver ourselves, instead of using the one that's built-in to the `OnboardingStep1V2Info`.  Our custom resolver pulls the onboarding info data out of the onboarding scaffolding area (migrating it from the old onboarding step 1 stash if needed) rather than directly from the onboarding step 1 stash (and same goes for step 3).

Reply:
ok, thanks for explaining!